cmd.go

```diff
@@ -655,7 +671,7 @@ func (u *ugbt) stdInfo(ctx context.Context) ([]info, error) {
 		return nil, err
 	}
 	if resp.StatusCode != http.StatusOK {
```

Comment:
IMHO the code probably wants to accept anything from 200 to 299, not just 200.

Reply:
The spec mandates 200 OK for a GOPROXY query success.